hw3/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "vite-react-starter",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "axios": "0.16.2",
+    "react": "18.3.1",
+    "react-dom": "18.3.1",
+    "react-router-dom": "6.23.1"
+  },
```

Comment:
I recommend using 'react-bootstrap' for navigation.
It comes with a prebuilt navigation component.
Here is a link to it: [link](https://react-bootstrap.github.io/docs/components/navbar).